src/hooks/use-deposit-status.ts

```diff
@@ -30,5 +38,24 @@ export function useDepositStatus(txId: string) {
     }
   }, [emilyUrl, transferTxStatus, txId]);
 
+  useEffect(() => {
+    const check = async () => {
+      const info = await getRawTransaction(txId);
+      if (info.status.confirmed) {
+        const currentBlockHeight = await getCurrentBlockHeight();
+        const unlockBlock =
+          Number(RECLAIM_LOCK_TIME || 144) + info.status.block_height;
+        const isPastLockTime = currentBlockHeight > unlockBlock;
+        if (isPastLockTime) {
+          setTransferTxStatus(DepositStatus.Failed);
+          clearInterval(interval);
+        }
```

Comment:
We tested this branch on devnet, it seems that we do get in here once the deposit is unlocked but the page/component/thingy doesn't refresh and stays as:
<img width="523" alt="image" src="https://github.com/user-attachments/assets/2809aa0b-a375-4bfb-b3b4-d4be480d2715" />
Refreshing the page we do get the updated view